test/test_utils/sagemaker.py

```diff
@@ -58,7 +58,7 @@ def assign_sagemaker_remote_job_instance_type(image):
 
 def assign_sagemaker_local_job_instance_type(image):
     if "graviton" in image:
-        return "c6g.2xlarge"
+        return "ml.c6g.2xlarge"
```

Comment:
SM local mode test should take regular instance type, not "ml" - SM local tests are running directly on EC2, not via sagemaker resources.

Reply:
Updated